docs/framework/winforms/controls/soundplayer-class.md

```diff
@@ -45,4 +45,4 @@ The `SoundPlayer` class enables you to easily include sounds in your application
  [Controls to Use on Windows Forms](../../../../docs/framework/winforms/controls/controls-to-use-on-windows-forms.md)  
  Provides a complete list of Windows Forms controls, with links to information on their use.  
   
- Also see [HYPERLINK "http://msdn.microsoft.com/library/11bxex12(v=vs.110)" My.Computer Object](https://msdn.microsoft.com/library/11bxex12\(v=vs.110\)) or [My.Computer Object](https://msdn.microsoft.com/library/11bxex12\(v=vs.120\)).
+ Also see [My.Computer Object](../../../visual-basic/language-reference/objects/my-computer-object.md) or [My.Computer Object](../../../visual-basic/language-reference/objects/my-computer-object.md).
```

Comment:
nit: "See also" is better than "Also see"
The two links are the same. One can be removed.